tensorrt_llm/_torch/auto_deploy/custom_ops/mamba/mamba_backend_common.py

```
@@ -49,7 +50,8 @@ def _mamba_ssm_prepare_metadata(
     Returns a tuple of (chunk_indices, chunk_offsets, seq_idx_prefill).
     """
     device = cu_seqlen.device
-    num_prefill, num_prefill_tokens, num_decode = batch_info_host.tolist()
+    batch_info = BatchInfo(batch_info_host)
+    num_prefill, num_prefill_tokens, num_decode = batch_info.get_absorbed_info()
```

Ruff (0.15.2)

[warning] Unpacked variable `num_prefill_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)